docs/process/issue-triage.md

```diff
@@ -98,11 +98,11 @@ issues from time to time that isn't and won't be covered here.
 
 ## More Information Needed
 
-Periodically we should be doing a sweep of issues that are open and labeled
-`more-information-needed`. If the original poster has not responded within
-two weeks after the last question by an official maintainer, close the issue
-with a message saying that you're closing the issue due to inactivity but are
-happy to re-open if they can provide more details.
+If we cannot understand or reproduce the issue with the information given, then we label it `more-information-needed`.
+
+Although we use a bot, we should still do a manual sweep of issues that are open and labeled `more-information-needed` at least once a week.
```

Comment:
This should be the responsibility of the first responder, right? If so, let's make that explicit:

💄

> Although we use a bot, the first responder should also do a manual sweep of issues that are open and labeled `more-information-needed` at least once a week.